src/main/java/Truck.java

```diff
@@ -0,0 +1,9 @@
+class Truck extends Machine {
+    public void doWork() {
+        System.out.println("Truck started working.");
+    }
```

Comment:
The message in `System.out.println` is not very informative. It should indicate that it is a Truck that is working. For example: `System.out.println("Truck is currently working.");`